src/app/project-detail/project-detail.component.html

```diff
@@ -3,17 +3,12 @@ <h2>{{ project.name }}</h2>
   <mat-divider></mat-divider>
 
   <div class="info">
-    <h3>Información</h3>
-    <!--
-    <p>Universidad: {{ project.universityName }}</p>
-    <p>Departamento: {{ project.department }}</p>
+    <h3>Información</h3>    
+    <p>Universidad: UTN</p> <!-- university nested needed -->
+    <!-- <p>Universidad: {{ project.department.university.name }}</p> -->
+    <p>Departamento: {{ project.department==undefined ? "Independiente" : project.department.name }}</p>
```

Comment:
In the next iteration use values returned from get one endpoint. . Note: include department.university in get one relations